artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/ServerConsumerImpl.java

```diff
@@ -918,7 +917,7 @@ public Queue getQueue() {
             }
 
             if (ref == null) {
-               ActiveMQIllegalStateException ils = ActiveMQMessageBundle.BUNDLE.consumerNoReference(id, messageID, messageQueue.getName());
+               ActiveMQIllegalStateException ils = new ActiveMQIllegalStateException("Cannot find ref to ack " + messageID);
```

Comment:
Was this on purpose? The former seemed more complete to me.

Reply:
This was not on purpose.